### PR TITLE
[MO] compress_to_fp16=False by default

### PR DESCRIPTION
**Pull request by pavel-esir**
### Details:
 - Large models utilize too much RAM on CPU when decompressed from f16->f32 before inference, therefore it's decided to return back to generate by default FP32 IRs.

### Tickets:
 - xxx-108268

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

File: docs/MO_DG/prepare_model/FP16_Compression.md [hunk 1]
```diff
@@ -2,18 +2,16 @@
...
+For most models, the accuracy drop is negligible.
 
-By default, models are compressed to ``FP16``, but you can disable compression by 
-specifying ``--compress_to_fp16=False``:
+To compress the model, use the `--compress_to_fp16` or `--compress_to_fp16=True` option:
 
 .. code-block:: sh
 
-   mo --input_model INPUT_MODEL --compress_to_fp16=False
+   mo --input_model INPUT_MODEL --compress_to_fp16
```
Comment (slyalin) [inline docs/MO_DG/prepare_model/FP16_Compression.md:14]: @kblaszczak-intel, please review these documentation changes. Any were else we need to provide adjustments?

Comment (kblaszczak-intel) [reply docs/MO_DG/prepare_model/FP16_Compression.md:14]: I've suggested a minor rewording but it does not change the sense. I can't find any other articles that would require adjusting to this issue.

**Comment by pavel-esir** [comment]
@kblaszczak-intel thanks for the suggestions! I added a couple of new sentences, please take a look. 
@rkazants also please review.

**Review by rkazants — COMMENTED** [review]
please correct info message:

```
def get_compression_message():
    link = "https://docs.openvino.ai/latest/openvino_docs_MO_DG_FP16_Compression.html"
    message = '[ INFO ] Generated IR will be compressed to FP16. ' \
              'If you get lower accuracy, please consider disabling compression explicitly ' \
              'by adding argument --compress_to_fp16=False.\n' \
              'Find more information about compression to FP16 at {}'.format(link)
    return message
```

**Review by rkazants — APPROVED** [review]

File: docs/MO_DG/prepare_model/FP16_Compression.md [hunk 2]
```diff
@@ -26,4 +24,11 @@ For details on how plugins handle compressed ``FP16`` models, see
    information about that.
 
 
+.. note::
+
+   Some large models (larger than a few Gb) when compressed to ``FP16`` may consume enormous amount of RAM on the loading
```
Comment (rkazants) [inline docs/MO_DG/prepare_model/FP16_Compression.md:29]: @pavel-esir, `Gb` -> `GB`. Please correct it in the next PR.

**Pull request merged**